server/test/unit/server/async/test_tasks.py

```diff
@@ -1059,3 +1059,12 @@ def test_is_not_worker_is_scheduler(self):
 
     def test_is_not_worker_is_resource_mgr(self):
         self.assertEquals(tasks._is_worker("resource_manager@some.hostname"), False)
+
+
+class TestPulpTask(unittest.TestCase):
+
+    def check_task_type(self):
```

Comment:
Can a docstring be added to this indicating that its purpose is to have all Celery tasks defined in Pulp inherit from PulpTask and not from Celery directly. Consider adding that it should not be adjusted unless you really know what you are doing. Normally I don't docstring tests, but this one is important.